cpp/cgi_progs/zotero_cgi.cc

```diff
@@ -323,6 +325,11 @@ HarvestTask::HarvestTask(const std::string &section, const std::string &output_f
     args.emplace_back("--output-filename=" + basename);
     args.emplace_back(ZTS_HARVESTER_CONF_FILE);
     args.emplace_back("JOURNAL");
+    if (not config_overrides.empty()) {
+        std::string arg("--config-overrides=" + config_overrides);
+        StringUtil::RemoveChars("\r", &arg);
+        args.emplace_back(arg);
```

Comment:
The 3 lines are nicer as a one-liner: args.emplace_back(StringUtil::RemoveChars("\n", "--config-overrides=" + config_overrides));